src/main/resources/requests/dailyCauseListRequests.ts

```diff
@@ -0,0 +1,41 @@
+import { dataManagementApi } from './utils/axiosConfig';
+
+const config = {
+  headers: {
+    verification: 'true',
+  },
+};
+
+export class DailyCauseListRequests {
+  public async getDailyCauseList(artefactId: string): Promise<any> {
+    try {
+      const response = await dataManagementApi.get('/publication/' + artefactId + '/payload', config);
+      return response.data;
+    } catch (error) {
+      if (error.response) {
+        console.log(error.response.data);
+      } else if (error.request) {
+        console.log(`Request failed. ${error.request}`);
+      } else {
+        console.log(`ERROR: ${error.message}`);
+      }
+    }
+    return null;
+  }
+
+  public async getDailyCauseListMetaData(artefactId: string): Promise<any> {
+    try {
+      const response = await dataManagementApi.get('/publication/' + artefactId, config);
+      return response.data;
+    } catch (error) {
+      if (error.response) {
+        console.log(error.response.data);
+      } else if (error.request) {
+        console.log(`Request failed. ${error.request}`);
+      } else {
+        console.log(`ERROR: ${error.message}`);
+      }
+    }
+    return null;
+  }
+}
```

Comment:
this isnt daily cause list requests, youre doing publication requests, if you make it look more generic than it actually is we'll end up with the same duplicated mutltiple times, see #278 , #277 or #220 for implementation of publication requests

Reply:
As discussed, code is using generic SummaryOfPublicationsService and PublicationRequest files.